shared/css/sidebar.css

```diff
@@ -3,96 +3,572 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /* XXX akita: Tidy up css */
-.media-wrapper:not(.showing-local-streams) > .media-control-buttons {
+/*------------------------------------*\
+    SIDEBAR
+\*------------------------------------*/
+
+body.sidebar {
+  height: 100%;
+}
+
+.sidebar-wrapper {
+  display: flex;
+  flex-flow: column wrap;
+  width: 250px;
+  height: 100%;
+}
+
+.sidebar-wrapper > .media-layout {
+  display: flex;
+  flex-flow: column wrap;
+  /* 188px video height + 40px media buttons height */
+  height: calc(188px + 40px);
+  position: relative;
+}
+
+.media-layout > .remote,
+.media-layout > .local {
+  /* to keep the 4:3 ratio set both height and width */
+  height: 188px;
+  width: 250px;
+  position: relative;
+  overflow: hidden;
+}
+
+.media-layout > .remote .remote-video {
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+}
+
+.media-layout > .remote {
+  /* Works around an issue with object-fit: cover in Google Chrome - it doesn't
+     currently crop but overlaps the surrounding elements.
+     https://code.google.com/p/chromium/issues/detail?id=400829 */
+  overflow: hidden;
+}
+
+.media-layout.showing-remote-streams > .local {
+  position: absolute;
+  z-index: 2;
+  padding: 8px;
+  right: 0;
+  left: auto;
+  bottom: 40px;
+  /* to keep the 4:3 ratio 80x60px + 16px padding + 4px border */
+  width: calc(80px + 16px + 4px);
+  height: calc(60px + 16px + 4px);
+}
+
+.media-layout.showing-remote-streams > .local > .remote-video-box {
+  border: solid 2px #fff;
+}
+
+.media-layout.showing-local-streams > .local > .remote-video-box > video {
+  width: 100%;
+  height: 100%;
+  /* Transform is to make the local video act like a mirror, as is the
+     convention in video conferencing systems. */
+  transform: scale(-1, 1);
+  transform-origin: 50% 50% 0;
+}
+
+
+/*------------------------------------*\
+    AVATAR
+    For any audio-only streams, we want
+    to display our own background
+\*------------------------------------*/
+
+.avatar {
+  background-image: url("../img/audio-call-avatar.svg");
+  background-repeat: no-repeat;
+  background-color: #4BA6E7;
+  background-size: contain;
+  background-position: center;
+  /*
+   * Expand to fill the available space, since there is no video any
+   * intrinsic width. XXX should really change to an <img> for clarity
+   */
+  height: calc(100% - 40px);
```

Comment:
What's the 40px for? It seems to adjust the height of the avatar when muted, so you get this strange resizing & border.